core/bind/core_bind.cpp

```diff
@@ -1300,6 +1308,9 @@ void _OS::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_low_processor_usage_mode_sleep_usec", "usec"), &_OS::set_low_processor_usage_mode_sleep_usec);
 	ClassDB::bind_method(D_METHOD("get_low_processor_usage_mode_sleep_usec"), &_OS::get_low_processor_usage_mode_sleep_usec);
 
+	ClassDB::bind_method(D_METHOD("set_low_priority_redraws_enabled", "enabled"), &_OS::set_low_priority_redraws_enabled);
+	ClassDB::bind_method(D_METHOD("are_low_priority_redraws_enabled"), &_OS::are_low_priority_redraws_enabled);
```

Comment:
Do we have setter methods that begin with `are` elsewhere? Maybe using `is` is better, even if it's less grammatically correct.

Reply:
Changed to `is_update_vital_only()`.